frontend/src/lib/components/LemonSelectWithSearch/LemonSelectWithSearch.scss

```diff
@@ -0,0 +1,91 @@
+.LemonSelectWithSearch {
+    .ant-select {
+        width: 100%;
+        .ant-select-selector,
+        &.ant-select-single .ant-select-selector {
+            padding: 0.25rem;
+            border-radius: var(--radius);
+            font-size: 0.875rem;
+            text-align: left;
+            line-height: 1.25rem;
+
+            border: 1px solid var(--border);
+            background: var(--bg-light);
+
+            .ant-select-selection-overflow {
+                gap: 0.25rem;
+            }
+        }
+
+        &:not(.ant-select-disabled):hover,
+        &.ant-select-focused:not(.ant-select-disabled),
+        &:not(.ant-select-disabled):active {
+            .ant-select-selector {
+                background: var(--bg-light);
+                border-color: var(--primary);
+                box-shadow: none;
+            }
+        }
+        &:not(.ant-select-disabled):active {
+            .ant-select-selector {
+                color: var(--primary-active);
+            }
+        }
+
+        .ant-select-selection-placeholder {
+            color: var(--text-muted);
+        }
+
+        &.ant-select-single {
+            .ant-select-selector {
+                height: 38px;
+                box-sizing: border-box;
+
+                .ant-select-selection-search-input {
+                    height: 38px;
+                }
+
+                .ant-select-selection-placeholder {
+                    padding-left: 0.4rem;
+                }
+            }
+        }
+
+        .ant-select-arrow {
+            display: none;
+        }
+    }
+}
+
+.LemonSelectWithSearchDropdown {
+    background: #fff;
+    padding: 0.5rem;
+    border-radius: var(--radius);
+    border: 1px solid var(--border);
```

Comment:
We're generally following a pattern where _actionable_ popovers have a `var(--primary)` border (e.g. compare insight tooltip vs. project switching dropdown)